tests/Command/RunCommandTest.php

```diff
@@ -784,57 +785,26 @@ public function testExecuteUnEncryptedJobData(): void
         self::assertEquals(0, $ret);
     }
 
-    public function testExecuteDoubleFailure(): void
```

Comment:
Pokryto v `testExecuteSkip`. Je ok že se to teď chová jinak?

Reply:
Přijde mi, že se to chová "stejně". Pointa je v tom, že i v případě chyby na postJobResult to skončí 0, protože se ta chyba zachytí. To byla pointa tohohle testu, takže, za mě ok.